### ISSUE TITLE
Data files for running the code in notebooks

### ISSUE THREAD

**Issue by urmeya** [tianlema/Multiview-AutoEncoder#1]
Can you please provide the data files like `combined2.pkl` and the data split indexes `data_split_idx` used in your notebooks?
```
data_split_idx_folder = f'{result_folder}/data_split_idx'

if not os.path.exists(project_folder):
  project_folder = 'F:/TCGA/Pan-Cancer-Atlas' # on my own desktop
filepath = f'{project_folder}/data/processed/combined2.pkl'`
```
It will be very helpful if you can provide the script files used to create this files from raw data.
Thank you in advance.

**Comment by tianlema** [comment]
@urmeya Thank you very much for your interest in this project. I did that project when I was a Ph.D. student. Due to changes in jobs and my computers, unfortunately, I no longer have the processed data for this project. However, the raw data are publicly available at the [NCI GDC data portal] (https://portal.gdc.cancer.gov/) and it is not difficult to process the data. 
I previously downloaded >1.5 TB RNA-seq, DNA methylation, protein expression, and other open data from the GDC data portal (the data portal has been continuously updated and they have more accurate data for more projects from recent releases). After downloading the data, I did some preprocessing and organized the data into multiple feature matrices, e.g., a gene expression matrix with rows corresponding to patient-samples and columns corresponding to genes, a miRNA expression matrix, a protein expression matrix, a methylation expression matrix, ... I aligned all the feature matrices so that each row in all the feature matrices corresponds to the same patient-sample. (It might need some effort to match all the clinical data with the multi-omic data directly from the GDC data portal.) Note I only select a subset of patient samples with more than one available -omic profiles. Each patient sample (some samples are actually non-diseased normal samples) also has clinical information, e.g., tumor stages. I selected a number of clinical variables and I basically tried to predict these clinical variables using multiple feature matrices (i.e., multi-views) in this paper. 

I recommend you read the autoencoder.py file as the model and the loss functions are defined there. 
Also, please note this repo is based on PyTorch 0.4.1. So there might be some compatibility issues with the current release of PyTorch.
